src/Fixer/Whitespace/IndentationTypeFixer.php

```diff
@@ -144,4 +144,19 @@ static function (array $matches) use ($indent) {
 
         return new Token([T_WHITESPACE, $newContent]);
     }
+
+    /**
+     * @param string $content
+     * @param string $indent
+     *
+     * @return string mixed
+     */
+    private function getExpectedIndent($content, $indent)
```

Comment:
There isn't **$this** context within the function. Can you use static declaration?
Call getExpectedIndent with **self** token within anonymous function for Preg::replaceCallback, will
prevent anonymous function relocation.

Reply:
static === global === evil
try to mock static method

no static here